### ISSUE TITLE
Invalid 2FA request

### ISSUE THREAD

**Issue by lukebp** [decred/politeiagui#2372]
I noticed that there are two issues when you navigate to the 2FA tab.

1. The gui sends a invalid request to politeiawww that returns a 400.  I don't think this request is needed at all.
2. The screen flickers a loading icon while this invalid request is being sent.

https://user-images.githubusercontent.com/9039877/117090323-ae02ad80-ad1d-11eb-8a5b-a169e776ca7f.mp4

**Comment by tiagoalvesdulce** [comment]
I don't think we can remove this request because it's the way we verify if the user has already set the TOTP. We hit `POST /v1/user/totp` and if we receive the Invalid params error (`code 77`) we know the user has enabled 2FA. We have some options to improve this:

1. Hit the endpoint in the background to prevent the flicker.
2. Return a `hasTotp` property on `GET /v1/user/me`.

**Comment by lukebp** [comment]
@tiagoalvesdulce I think it makes more sense to include this in the `/user/me` response.  That data will already be cached in the gui, right?  So a extra request would not be required when navigating to the 2fa page.

**Comment by tiagoalvesdulce** [comment]
That's right
